### PR TITLE
Badger red accessibility

### PR DESCRIPTION
**Pull request by dstodolny**
### Motivation

- [Related story](https://github.com/redbadger/website-honestly/issues/431)

owkt3tf

### Pre-merge checklist

- [ ] Documentation
- [ ] Unit tests
- [ ] Reviews
  - [ ] Code review
  - [ ] Design review
- [ ] Manual testing
  - [ ] Windows 7 IE 11
  - [ ] Windows 10 Edge
  - [ ] Mac OS El Capitan Safari
  - [ ] Mac OS El Capitan Chrome
  - [ ] Mac OS El Capitan Firefox
  - [ ] iOS 9 Safari
  - [ ] Android 6 Chrome
  - [ ] Listen to the site on a screen reader
  - [ ] Navigate the site using the keyboard only
- [ ] Tester approved

### CODE REVIEW

File: site/css/_colors.css [hunk 1]
```diff
@@ -9,7 +9,8 @@
 */
 
 /* Main Colour Palette */
-@value badgerRed: #FC1D42;
+@value badgerRed: #FC3A54;
+@value badgerRedOnBlack: #E5051A;
```
Comment (gregdorward) [inline site/css/_colors.css]: badgerRed and badgerRedOnBlack values need to be switched throughout the site - my mess up in the comment on the original bug mixing these up

**Comment by gregdorward** [comment]
@dstodolny, I think there's a couple of changes needed to this one, having seen the new colours implemented.

Firstly, I think we need a third red colour to re-implement the original Badger Red (#FC1D42) which needs to be applied to any buttons throughout the site. The text within these buttons should be large enough that these pass contrasting guidelines, and it looks more consistent with the RB logo

Secondly, could we make the elements within the ul class 'style__categories___ilyar' on the Meet Our Team page Badger Red (#FC1D42), but also increase the font size of these to 18px please? Let me know if you have any questions

**Comment by gregdorward** [comment]
This is looking really good to me. I'll catch up with Sari to show her tomorrow for final approval

**Comment by gregdorward** [comment]
Added a 'Don't merge' label temporarily, as we ideally want to get this out alongside the blog colour change for consistency across the site

**Comment by gregdorward** [comment]
@dstodolny this PR still hasn't been merged if you wanted to do the 18px text change as part of this one?

**Comment by gregdorward** [comment]
After chatting to Sari, we've decided to keep the font size as is for tablet and mobile, and increase to 18.66 for desktop

Also, in order to pass accessibility checks, we need to specify that ProximaNova-Bold has bold weighting, seeing as it's a custom font

**Comment by gregdorward** [comment]
@dstodolny it looks like the new Q and A section still has the old Badger Red colour for the category titles - class="style__category__title___3777I _fonts__fontXS___2mdTk _fonts__boldSansSerif___1G6P5"

Would you be able to update them to use #E5051A please?

**Comment by dstodolny** [comment]
We should merge this PR first and then change the colour in the `badger-red-accessibility`. Otherwise we will produce conflicts in these PRs.

**Comment by dstodolny** [comment]
We should first merge the `history-slice` branch, rebase the `master` branch and then make these changes here.

**Comment by gregdorward** [comment]
This one looks good to go, so will push it once the blog part is done as well

**Pull request merged**